pyiron/interactive/scipy_minimizer.py

```diff
@@ -52,13 +40,6 @@ def set_input_to_read_only(self):
     def write_input(self):
         pass
 
-    def interactive_close(self):
```

Comment:
Why is this part removed?

Reply:
It didn't change anything with or without this part.

Reply:
Is the output in the HDF5 file the same? in particular in the generic section?

Reply:
yes

Reply:
Actually there was anyway nothing in `generic`

Reply:
Ok, it seems like `interactive_is_activated()` is not even defined so this was never working ... Still the intention was that the minimiser running in interactive mode could minimize multiple structures which are given to it one by one. But I guess that is a separate pull request.